File: JavaBasics/ForLoops/Exercise/Oscars.java
package EntryModule.ForLoops.Exercise;

import java.util.Scanner;
public class Oscars {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String name = scanner.nextLine();
        double academypts = Double.parseDouble(scanner.nextLine());
        int number  = Integer.parseInt(scanner.nextLine());



        for(int i=0; i<number; i++){
            String nametime = scanner.nextLine();
            double pts = Double.parseDouble(scanner.nextLine());
            double points=pts*nametime.length();
            academypts=academypts+(points/2);
            if(academypts>=1250.5){
                System.out.printf("Congratulations, %s got a nominee for leading role with %01.1f!", name, academypts);
                break;
            }
        }
        if(academypts<1250.5){
            System.out.printf("Sorry, %s you need %01.1f more!", name, (1250.5-academypts));
        }




    }
}

File: JavaScriptBasics/ForCycle/Exercise/Oscars.js
function Oscars(input){

let name = input[0];
let academypoints = Number(input[1]);
let juryNumber = Number(input[2]);

for(let i = 3; i<(juryNumber*2)+3; i=i+2){
    let juryMemberName = input[i];
    let juryMemberPoints = Number(input[i+1]);
    academypoints=academypoints+((juryMemberPoints*juryMemberName.length)/2);
    if(academypoints>=1250.5){
        console.log(
            "Congratulations, "+name+" got a nominee for leading role with "+ academypoints+"!"
        );break;
    }
}
if(academypoints<1250.5){
    console.log("Sorry, "+name+" you need "+(1250.5-academypoints)+" more!");
}


}